### PR TITLE
Updated date range arrow to single character #1058

### PR DESCRIPTION
**Pull request by DeviousLab**
#### Summary
Switches the arrow in a date range from `->` to `→` as requested
image

#### Ticket Link
  Aims to resolve https://github.com/mattermost/focalboard/issues/1058

### CODE REVIEW

**Review by hahmadia — APPROVED** [review]

**Comment by hahmadia** [comment]
Hi! Could you please fix the snapshot test that are failing? Thank you

**Comment by DeviousLab** [comment]
https://github.com/mattermost/focalboard/blob/e8eeb5104670d94bb31cb2ab4a82cc5d0a75dafe/webapp/src/components/properties/dateRange/dateRange.test.tsx#L158

https://github.com/mattermost/focalboard/blob/e8eeb5104670d94bb31cb2ab4a82cc5d0a75dafe/webapp/src/components/properties/dateRange/dateRange.test.tsx#L195

https://github.com/mattermost/focalboard/blob/e8eeb5104670d94bb31cb2ab4a82cc5d0a75dafe/webapp/src/components/properties/dateRange/dateRange.test.tsx#L230

Am I correct in assuming that these are the lines causing Jest to fail?

**Review by harshilsharma63 — APPROVED** [review]
Looks good but we need to update the tests.

**Comment by kamre** [comment]
@DeviousLab here are the [log messages about failures](https://github.com/mattermost/focalboard/pull/1064/checks?check_run_id=3400213587#step:11:24). So it looks like you need to fix two things:
* update jest snapshots via executing `npm run updatesnapshots` from `webapp` folder
* update names inside these three calls of `getByRole` that you have already found

**Comment by DeviousLab** [comment]
Ran `npm run updatesnapshots` after updating the names in dateRange.test.tsx and all tests passed.

**Review by hahmadia — APPROVED** [review]

**Comment by DeviousLab** [comment]
> @DeviousLab did you forget to commit the updated snapshots?

My bad, wasn't aware that they were necessary to commit!

File: webapp/src/components/cardDetail/__snapshots__/cardDetailContents.test.tsx.snap [hunk 1]
```diff
@@ -597,19 +597,19 @@ exports[`components/cardDetail/cardDetailContents should match snapshot after on
         class="octo-block-margin"
       />
       <div
-        class="MarkdownEditor octo-editor  active"
+        class="MarkdownEditor octo-editor  "
```
Comment (hahmadia) [inline webapp/src/components/cardDetail/__snapshots__/cardDetailContents.test.tsx.snap:600]: I don't think this file needs change. I think you can uncommit the changes in this file.

**Review by hahmadia — CHANGES_REQUESTED** [review]

**Comment by DeviousLab** [comment]
Removed the other file from the commit now

**Review by hahmadia — APPROVED** [review]

**Pull request merged**